Makefile.in

```diff
@@ -4,19 +4,23 @@ AS		?= $(CC)
 AR		= ar
 RM		= rm -f
 
-CFLAGS	+= -Wall -Werror -Wextra -fpic -fno-builtin -fno-stack-protector -nostdlib \
-			 -nostdinc -Iinclude
+BASE_CFLAGS	+= -Wall -Werror -Wextra
+LIBC_CFLAGS = $(BASE_CFLAGS) -Iinclude -nostdlib -nostdinc -fno-stack-protector -fno-builtin -fpic
+SYSDEPS_CFLAGS += $(BASE_CFLAGS) -fno-stack-protector -fpic
+
 ifneq ($(CC), tcc)
-CFLAGS	+= -ansi -pedantic
+BASE_CFLAGS	+= -ansi -pedantic
```

Comment:
Just to ask
why aren't we using `-ansi -pedantic` only on TCC and not everywhere ? :thinking:

Reply:
No. The flags are used only when **not** using TCC. This is most likely because tcc doesn't support those flags